src/Package/Impl/DataInspect/VariableGridHost.xaml.cs

```diff
@@ -54,7 +54,7 @@ public partial class VariableGridHost : UserControl {
         internal void SetEvaluation(VariableViewModel wrapper) {
             VsAppShell.Current.AssertIsOnMainThread();
 
-            if (wrapper.TypeName == "NULL" && wrapper.Value == "NULL") {
+            if (wrapper.TypeName == null && wrapper.Value == null) {
```

Comment:
If I set an existing variable to `NULL` or `null` (same result):
```
ip <- NULL
```
then TypeName is `"NULL"` and Value is `null`.  It goes in the else statements and the result is a blank grid.  So that's fine.